### PR TITLE
Mapping progress update (OMIM): following angle bracket bugfix

### PR DESCRIPTION
**Pull request by joeflack4**
Updates
```
    - Update: Mapping progress monitor: Re-ran all mapping progress artefacts and documentation, following a bugfix where terms were not correctly being parsed due to an <angle brackets> bug
```

Also includes:
- #250

### CODE REVIEW

File: src/ontology/reports/omim_mapping_status.tsv
Comment (joeflack4) [inline src/ontology/reports/omim_mapping_status.tsv:376]: @matentzn @sabrinatoro @nicolevasilevsky After the bugfix in #250 and some help from @hrshdhgd , I recreated all my SemanticSQL DB files and then re-ran the progress monitor. It looks like only OMIM had updates.

I'm kind of surprised the OMIMPS terms weren't there before. I think I had added the prefix for OMIMPS, but I didn't notice this bug until now, so they weren't getting picked up until I fixed it. Kinda surprised the bug only affected OMIM though.

Comment (matentzn) [reply src/ontology/reports/omim_mapping_status.tsv:376]: Ok, thanks!

**Review by matentzn — APPROVED** [review]

**Comment by joeflack4** [comment]
Holding off on merging this pending approval of #250, which this PR also includes.

**Pull request merged**
